spec/services/reports/agency_invoice_iaa_supplement_report_spec.rb

```diff
@@ -9,11 +9,75 @@
     end
 
     context 'with data' do
+      let(:partner_account1) { create(:partner_account) }
+      let(:partner_account2) { create(:partner_account) }
+      let(:gtc1) do
+        create(
+          :iaa_gtc,
+          gtc_number: 'gtc1234',
+          partner_account: partner_account1,
+          start_date: iaa1_range.begin,
+          end_date: iaa1_range.end,
+        )
+      end
+
+      let(:gtc2) do
+        create(
+          :iaa_gtc,
+          gtc_number: 'gtc5678',
+          partner_account: partner_account2,
+          start_date: iaa2_range.begin,
+          end_date: iaa2_range.end,
+        )
+      end
+
+      let(:iaa_order1) {
+        build_iaa_order(order_number: 1,  date_range: iaa1_range, iaa_gtc: gtc1)
+      }
+      let(:iaa_order2) {
+        build_iaa_order(order_number: 2,  date_range: iaa2_range, iaa_gtc: gtc2)
+      }
+
+      # Have to do this because of invalid check when building integration usages
+      let!(:iaa_orders) do
+        [
+          iaa_order1,
+          iaa_order2,
+        ]
+      end
+
+      let(:integration1) {
+ build_integration(issuer: iaa1_sp.issuer, partner_account: partner_account1)
+      }
+      let(:integration2) {
+ build_integration(issuer: iaa2_sp.issuer, partner_account: partner_account2)
+      }
```

Comment:
I think Rubocop is going to be angry at this... `do...end` for multiline block and also indentation
```suggestion
      let(:integration1) do
        build_integration(issuer: iaa1_sp.issuer, partner_account: partner_account1)
      end
      let(:integration2) do
        build_integration(issuer: iaa2_sp.issuer, partner_account: partner_account2)
      end
```